mne/simulation/source.py

```diff
@@ -218,3 +220,121 @@ def simulate_stc(src, labels, stc_data, tmin, tstep, value_fun=None):
 
     stc = SourceEstimate(data, vertices=vertno, tmin=tmin, tstep=tstep)
     return stc
+
+
+def make_forward_dipole(dipole, bem, info, trans=None, n_jobs=1, verbose=None):
```

Comment:
OK with me. Shall I go ahead with the move, modify tests accordingly and upload a few examples?